test/JDBC/input/switchoffset.sql

```diff
@@ -0,0 +1,138 @@
+Select switchoffset('2001-04-22 ', -120)
```

Comment:
Test cases with null as one or both arguments
Instead of text, pass other data types for both arguments
Input as table column and part of expressions as both input/output

Reply:
Added